### PR TITLE
Update file owners

### PR DESCRIPTION
**Pull request by lucasjohnston**
Update CODEOWNERS file, so the relevant framework or code owners review any suggested changes made.

Would like an approval from an emma on the frameworks side, and from liam on the web-approvers side :)

### CODE REVIEW

**Comment by emmanorthcott** [comment]
Only the framework owner should be able to approve content changes to frameworks in order to provide consistency. If someone is going away for an extended period of time (for example +2weeks) they should delegate the responsibilities of framework owner to a deputy.

**Comment by lucasjohnston** [comment]
Okay, nice. In that case I think we can implement this

File: CODEOWNERS [hunk 1]
```diff
@@ -1,18 +1,40 @@
...
-CODEOWNERS  @monzo/web-approvers
-package.json  @monzo/web-approvers
-yarn.lock  @monzo/web-approvers
-*.js  @monzo/web-approvers
-*.json  @monzo/web-approvers
-
 # These owners will be the default owners for everything in the repo.
-*       @lucasjohnston
\ No newline at end of file
+*       @lucasjohnston
```
Comment (liamn) [inline CODEOWNERS]: I wonder if we should have this going forward? It seems you'd be covered for most technical stuff by the web-approvers group?

Comment (lucasjohnston) [reply CODEOWNERS]: By being at the top, the * is effectively a last resort - it means this `CODEOWNER` will only be requested if the changes do not match any other rule (which should never happen) - I'll probably change this to web-approvers or the whole org in the future, but at the mo I think it's fine to be me

Comment (lucasjohnston) [reply CODEOWNERS]: So I won't be tagged in everything 😛

**Pull request merged**